views/components/Graphs/GraphTable.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import {
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Table,
+  Paper,
+  Toolbar,
+  Typography,
+} from '@material-ui/core'
+
+const GraphTable = ({ graph }) => {
+  return (
+    <Paper>
+      <Toolbar>
+        <Typography variant="h6" color="inherit">
+          {graph?.title.toUpperCase()} TABLE
+        </Typography>
+      </Toolbar>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell align="center">Site</TableCell>
+            {graph?.labels?.map(({ name }) => (
+              <>
+                <TableCell key={name + 'count'} align="center">
+                  {name}
+                </TableCell>
+                <TableCell key={name + 'percent'} align="center">
+                  {name}%
+                </TableCell>
+              </>
+            ))}
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {graph?.dataBySite
+            .slice(0)
+            .reverse()
```

Comment:
why do we need to slice and reverse?

Reply:
gotcha. I _think_ you could solve that with something like

```js
const [totalsSite, ...sites] = graph.dataBySite
```

And then iterate through `sites` and put `totalsSite` at the end.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/175/commits/6eaaa4b6b4bfc4375ce6e4251bede79d0598f08b